cmd/gearlogger/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"net/url"
+	"time"
+
+	"github.com/Clever/gearadmin"
+	"gopkg.in/Clever/kayvee-go.v1"
+)
+
+func logMetrics(g gearadmin.GearmanAdmin) {
+	statuses, err := g.Status()
+	if err != nil {
+		log.Fatalf("error retrieving gearman status: %s", err)
+	}
+	for _, status := range statuses {
+		fmt.Println(kayvee.FormatLog("gearlogger", "info", "status", map[string]interface{}{
+			"function": status.Function,
+			"running":  status.Running,
+			"total":    status.Total,
+			"workers":  status.AvailableWorkers,
+		}))
```

Comment:
I'm going to make a Dockerfile that runs this. Why is `stderr` preferred over `stdout`? Seems like for something that is expected output, and not debug info, `stdout` seems more "normal".